agent/api/ecsclient/client_test.go

```diff
@@ -269,6 +292,20 @@ func TestRegisterContainerInstance(t *testing.T) {
 	}
 }
 
+func TestRegisterInstanceBadAttribute(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	err := awserr.New("ClientException", "Invalid attribute given", errors.New("Invalid attribute given"))
```

Comment:
I don't understand what this test is asserting...you're just creating an error and then immediately validating that it's the error you expect.